nodespec/aws.go

```diff
@@ -0,0 +1,196 @@
+// Package nodespec gives us access to provider-specific node specification.
+package nodespec
+
+import (
+	"github.com/giantswarm/microerror"
+	"gopkg.in/yaml.v2"
+)
+
+var (
+	// specYAML is the raw data on all necessary AWS instance types taken from
+	// https://github.com/giantswarm/installations/blob/master/default-draughtsman-configmap-values.yaml
+	// Warning: YAML in Golang is super fragile. There must not be any tabs in this string, otherwise
+	// the marshalling will fail. However we will likely detect this in CI when running tests.
+	awsInstanceTypesYAML = `
```

Comment:
So here is my confusion - in the test you wrote above there is memory and storage provided. This should still work for all providers because it comes from `cluster-service` like before. What am I missing?